htdocs/fichinter/card.php

```diff
@@ -1766,6 +1779,29 @@
 		$params = array();
 		if ($user->socid == 0) {
 			if ($action != 'editdescription' && ($action != 'presend')) {
+				// Subtotal
+				if ($object->status == Fichinter::STATUS_DRAFT && isModEnabled('subtotals') && getDolGlobalString('SUBTOTAL_TITLE_'.strtoupper($object->element))) {
+					$langs->load('subtotals');
+
+					$url_button = array();
+					$url_button[] = array(
+						'lang' => 'subtotals',
+						'enabled' => (isModEnabled('ficheinter') && $object->status == Fichinter::STATUS_DRAFT),
```

Comment:
Replace with 
isModEnabled('intevention')

Reply:
made it with the good name,
The intervention module is complex to maintain due to its technical debt of different terms (interventional, fichinter, ficheinter, etc.). Perhaps it's time we took some time to harmonize all of this?